generators/app/USAGE

```diff
@@ -0,0 +1,7 @@
+Description:
```

Comment:
Yes, I'm not sure if it's only Yeomans conventions, but that how it looked in examples, and that's how it works when you do 
`yo ventures --help`

Reply:
<img width="1040" alt="Screenshot 2022-01-12 at 21 08 08" src="https://user-images.githubusercontent.com/5908855/149214064-c51d7a1b-91c7-4473-b0d6-49c7f63c2a15.png">

This is what the console renders for me as the help. However doesn't really makes sense for me. Especially

Usage:
`yo ventures:app [options]` I guess app is `dotfiles|github` but is not clear, since it's team tool I would make it clearer if possible. 

Example:
Should be `yo ventures` right? In the help is written just `ventures`

Reply:
`yo ventures:app [options]` is generated by Yeoman, as there is a convention that `generatorfoo:app` is the default = `generatorfoo`. I'd take a look if we could change that, but could be tricky.

> I guess app is `dotfiles|github`
app could be nothing == `app`, `dotfiles`, or `github`

You can check the `yo ventures:github --help` as well.

> Example:
> Should be yo ventures right? In the help is written just ventures

Yes, I'm going to change that.

Reply:
WDYT of 
```
Usage:
  yo ventures[:app] [options]
```
Added in https://github.com/tomalec/generator-ventures/pull/1/commits/df45a7b94769cb26068af304a273048718347d93

Reply:
That looks clearer (for me at least). Thanks for changing it